etc/kayobe/kolla.yml

```diff
@@ -126,9 +126,8 @@ kolla_build_blocks:
     RUN grafana-cli plugins install vonage-status-panel
     # NOTE(brtknr): Install upstream patch currently under review:
     # https://review.opendev.org/#/c/670316/
-    RUN git clone https://github.com/stackhpc/monasca-grafana-datasource --depth=1 -b stackhpc/1.3.0 \
-        && grafana-cli --pluginsDir monasca-grafana-datasource/ plugins install monasca-datasource \
-        && rm -rf monasca-grafana-datasource/
+    RUN grafana-cli plugins remove monasca-datasource
+        && git clone https://github.com/stackhpc/monasca-grafana-datasource --depth=1 -b stackhpc/1.3.0 /var/lib/grafana/plugins/monasca-datasource/ \
```

Comment:
I'm not sure I understand why we are changing this. The original code uses the Grafana CLI to install the `plugin, and it should go to the `/var/lib/grafana/plugins/` directory, which is essentially the same as the new method except that doesn't use the Grafana CLI?

Reply:
@brtknr ^ Any thoughts on this?

Reply:
The reason we're changing this is because the original code was not doing what I thought it was doing... the function of the `--pluginsDir` option is in fact to redirect where the plugin is saved to rather than specifying the source of the plugin. The documentation in fact recommends that if you are manually installing a plugin, you should instead manually download the plugin in the plugin directory specified in the `grafana.ini` config file: https://grafana.com/docs/plugins/installation/#installing-plugins-manually. Since the `--pluginUrl` option doesn't exist in the version of Grafana we are running, unpacking it into the given folder seems to do the trick.

Reply:
Thanks, I see now. The advantage of the CLI is that if someone changed the plugin folder it would automatically put it in the right place, but as you say it's not supported in 4.2 [1] so this is fine. Please could you add your comment to the commit message?

[1] https://grafana.com/docs/v4.2/plugins/installation/